modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZTextWithMediaParagraphBehavior.php

```diff
@@ -179,9 +177,6 @@ public function preprocess(&$variables) {
     }
     $paragraph_status = $paragraph->status->value ? 'published' : 'unpublished';
     $variables['attributes']['id'] = HTML::getId($paragraph->bundle() . '-' . $paragraph->id());
-    if (!empty($variables['attributes']) && !empty($variables['attributes']['class']) && !is_array($variables['attributes']['class'])) {
-      $variables['attributes']['class'] = [];
-    }
     $variables['attributes']['class'][] = 'paragraph';
```

Comment:
We could also do something like this to ensure `$variables['attributes']['class']` is an array without blowing away its existing value:

```suggestion
    if (!empty($variables['attributes']) && !empty($variables['attributes']['class']) && !is_array($variables['attributes']['class'])) {
      $variables['attributes']['class'] = [$variables['attributes']['class']];
    }
    $variables['attributes']['class'][] = 'paragraph';
```